spdx/parsers/lexers/tagvalue.py

```diff
@@ -95,7 +96,11 @@ def t_text_end(self, t):
         t.type = 'TEXT'
         t.value = t.lexer.lexdata[
             t.lexer.text_start:t.lexer.lexpos].strip()
-        t.lexer.lineno += t.value.count('\n')
+        try :
+        	unicode
+        	t.lexer.lineno += t.value.count('\n'.encode("utf-8"))
```

Comment:
Actually tagvalue.py in parsers has a lot of decode options, which was raising error while opening the file with `codecs`. So I plan to not change the current test ( openfile with `open` ) and change the code only once here.  
Default encoding is ascii in `open`, so I have to specifically encode it to utf-8. This is needed to pass the test I think.